Source/VCGeneration/Checker.cs

```diff
@@ -417,7 +423,7 @@ public override void BeginCheck(string descriptiveName, VCExpr vc, ErrorHandler
     }
 
     [NoDefaultContract]
-    public override Outcome CheckOutcome(ErrorHandler handler, int taskID = -1)
+    public override Outcome CheckOutcome(ErrorHandler handler, int errorLimit)
```

Comment:
?

Reply:
In both places where this function is called, and in its parent class original definition, the second argument is named "errorLimit" and not a "taskID", so this seems like is a discrepancy I fixed here.